apps/team-awesome-web/src/types/teamMember.ts

```diff
@@ -1,4 +1,5 @@
 export type TeamMember = {
+  __typename?: string;
```

Comment:
What does __ do?

Reply:
`__typename` is used in GraphQL queries to indicate what type the fetched object is: I attempted to use this logic to automatically assign the dropdown arrays in the Select component. Will remove this as it's unused.